Dockerfile

```diff
@@ -11,21 +11,11 @@ RUN go mod download
 
 COPY . ./
 
-RUN go build -o /app/mochi ./cmd
-
+RUN go build -o /app/mochi ./cmd/docker
 
 FROM alpine
 
 WORKDIR /
 COPY --from=builder /app/mochi .
 
-# tcp
-EXPOSE 1883
```

Comment:
I removed `EXPOSE` for a few reasons:
`EXPOSE` is not functional and may serve to confuse users as with configuration opens available via file these can be changed to anything.